MonoGame.Framework/MacOS/MacGamePlatform.cs

```diff
@@ -380,6 +380,13 @@ public override void ResetElapsedTime()
         {
             _gameWindow.ResetElapsedTime();
         }
+		
+		public override void Present()
+        {
+            var device = Game.GraphicsDevice;
+            if (device != null)
+                device._graphicsMetrics = new GraphicsMetrics();

```

Comment:
This is pretty gross... but we plan to kill the `MacGamePlatform` in favor of `OpenTKGamePlatform` soon anyway.